hack/undeploy-karmada.sh

```diff
@@ -44,6 +44,25 @@ ETCD_HOST_IP=$(kubectl get pod -l app=etcd -n karmada-system -o jsonpath='{.item
 # clear all in namespace karmada-system
 kubectl delete ns karmada-system --kubeconfig="${HOST_CLUSTER_KUBECONFIG}"
 
+# clear global resources
+# delete clusterroles
+function delete_cluster_roles() {
+  for role in `kubectl  get clusterrole --kubeconfig="${HOST_CLUSTER_KUBECONFIG}" | grep karmada-controller-manager |  awk '{print $1}' `
+  do
+    kubectl delete clusterrole $role --kubeconfig="${HOST_CLUSTER_KUBECONFIG}"
+  done
+}
+delete_cluster_roles
+
+# delete clusterrolebindings
+function delete_cluster_role_bindings() {
+  for rolebinding in `kubectl  get clusterrolebindings --kubeconfig="${HOST_CLUSTER_KUBECONFIG}" | grep karmada-controller-manager |  awk '{print $1}' `
```

Comment:
Why are loops used here？Are there multiple `ClusterRole` resources that the name including `karmada-controller-manager`?

Reply:
Yes 
 But I do not know when the clusterrole  created
<img width="909" alt="image" src="https://user-images.githubusercontent.com/16109961/166624728-9e2c3140-3b5f-4d4d-bbfc-371263a34584.png">

Reply:
Right, I guess you registered `karmada-host` into control plane. And the namespace `karmada-cluster` in `karmada-host`  hasn't been removed.

Reply:
Yes，and how should I modify the clear cluster role process

Reply:
I think we only remove all resources created by the control plane in `undeploy-karmada.sh`.

Reply:
I got it

Reply:
please help me review again